datafusion/sql/src/unparser/expr.rs

```diff
@@ -618,6 +681,50 @@ impl Unparser<'_> {
         }
     }
 
+    // TODO: operator precedence should be defined in sqlparser
```

Comment:
This code is actually using the sql parser precidence, right? So we could perhaps remove this todo comment

Reply:
it maps sqlparser operators to datafusion_expr::Operators first as sqlparser operators don't have precedence defined. That's why I added the sql_to_op below